src/api/apiType.ts

```diff
@@ -0,0 +1,33 @@
+export interface articles {
+  totalCount?: number;
+  list: [
+    {
+      updatedAt: string;
+      createdAt: string;
+      likeCount: number;
+      writer: {
+        nickname: string;
+        id: number;
+      };
+      image: string | null;
+      content: string;
+      title: string;
+      id: number;
+    }
+  ];
+}
```

Comment:
```suggestion
export interface articles {
  totalCount?: number;
  list: {
      updatedAt: string;
      createdAt: string;
      likeCount: number;
      writer: {
        nickname: string;
        id: number;
      };
      image: string | null;
      content: string;
      title: string;
      id: number;
    }[];
}
```
위처럼 코드를 작성하면 배열안에 첫번쨰인덱스만 할당 가능한 튜플형태로 타입이 지정돼요! 제가 올린것처럼 사용하시면 배열로 쓰실 수 있어요~